src/libechonest.mk

```diff
@@ -0,0 +1,28 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := libechonest
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 2.3.1
+$(PKG)_CHECKSUM := ab961ab952df30c5234b548031594d7e281e7c9f2a9d1ce91fe5421ddde85e7c
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/lfranchi/libechonest/archive/$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc qt qjson
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'https://github.com/lfranchi/libechonest/archive/' | \
+    $(SED) -n 's,.*/\([0-9][^"]*\)/"\.tar.*,\1,p' | \
+    sort | uniq | \
+    head -1
+endef
+
+define $(PKG)_BUILD
+    mkdir '$(1)/build'
+    cd '$(1)/build' && cmake .. \
+        -DCMAKE_TOOLCHAIN_FILE='$(CMAKE_TOOLCHAIN_FILE)'
+
+    $(MAKE) -C '$(1)/build' -j '$(JOBS)' install
+endef
+
+$(PKG)_BUILD_STATIC =
```

Comment:
It compiles statically in Linux machine after replacing `SHARED` with `STATIC`. After applying [the patch](https://gist.github.com/f0d4a8e53640f0b04c11) MXE shared target builds as before and static target [fails](https://gist.github.com/20b64c4141921b8f25dc) with dllimport linking errors. Can you fix them, please? Maybe we should fix this in upstream also.

@tonytheodore, can you see [CMake warning](https://gist.github.com/starius/20b64c4141921b8f25dc#file-libechonest_i686-w64-mingw32-static-L46) from the build. CMake modules of MXE seem to be installed to multiple directories and conflict with themselves.